index.html

```diff
@@ -15,6 +15,9 @@
 2nd Number: <input type="text" id="secondNumber" /><br>
 <input type="button" onClick="multiplyBy()" Value="Multiply" />
 <input type="button" onClick="divideBy()" Value="Divide" />
+<input type="button" onClick="addition()" Value="Addition" />
```

Comment:
Here it would be more efficient and demonstrate cleaner code for a drop down box with you operators in them.